app/src/main/java/com/orange/ods/app/ui/components/listitem/ComponentListItem.kt

```diff
@@ -0,0 +1,236 @@
+        OdsChoiceChip(textRes = R.string.component_list_leading_wide_image, value = ListItemCustomizationState.Leading.WideImage)
+    }
+
+    Subtitle(textRes = R.string.component_list_trailing, horizontalPadding = true)
+    OdsChoiceChipsFlowRow(
+        selectedChip = listItemCustomizationState.selectedTrailing,
+        modifier = Modifier.padding(horizontal = dimensionResource(id = R.dimen.screen_horizontal_margin)),
+        outlinedChips = true
+    ) {
+        listItemCustomizationState.trailings.forEach { trailing ->
+            OdsChoiceChip(textRes = trailing.textResId, value = trailing)
+        }
+    }
+
+}
+
+@Composable
+private fun ComponentListsContent(listItemCustomizationState: ListItemCustomizationState) {
+    val recipes = LocalRecipes.current.filter { it.description.isNotBlank() }
+    with(listItemCustomizationState) {
+        Column(modifier = Modifier.verticalScroll(rememberScrollState())) {
+            if (!trailings.contains(selectedTrailing.value)) {
+                resetTrailing()
+            }
+
+            val modifier = Modifier
+                .let { modifier ->
+                    iconType?.let { modifier.iconType(it) }.orElse { modifier }
+                }
+            val singleLineSecondaryText = lineCount.value == 2
+
+            val text = recipes.first().title
+            val secondaryText = listItemCustomizationState.getSecondaryText(recipes.first())
+            val icon: @Composable (OdsListItemIconScope.() -> Unit)? =
+                listItemCustomizationState.getIconPainter(recipes.first())?.let { { OdsListItemIcon(painter = it) } }
+            trailing?.let { listItemTrailing ->
+                OdsListItem(
+                    modifier = modifier,
+                    text = text,
+                    secondaryText = secondaryText,
+                    singleLineSecondaryText = singleLineSecondaryText,
+                    icon = icon,
+                    trailing = listItemTrailing
+                )
+            }.orElse {
+                val context = LocalContext.current
+                OdsListItem(
+                    modifier = modifier.clickable { clickOnElement(context = context, context.getString(R.string.component_list_item)) },
+                    text = text,
+                    secondaryText = secondaryText,
+                    singleLineSecondaryText = singleLineSecondaryText,
+                    icon = icon
+                )
+            }
+            CodeImplementationColumn(modifier = Modifier.padding(horizontal = dimensionResource(id = R.dimen.screen_horizontal_margin))) {
+                FunctionCallCode(
+                    name = OdsComposable.OdsListItem.name,
+                    parameters = {
+                        string("text", text)
+                        if (singleLineSecondaryText) secondaryText?.let { string("secondaryText", it) }
```

Comment:
`secondaryText` parameter should be displayed if it is not null, you don't need to check `singleLineSecondaryText`.

Moreover, `singleLineSecondaryText` parameter is missing.

Here is the new parameters block:

```kotlin
parameters = {
    string("text", text)
    if (secondaryText != null) {
        string("secondaryText", secondaryText)
        if (!singleLineSecondaryText) stringRepresentation("singleLineSecondaryText", false)
    }
    icon()
}
```